backend/src/services/GenerateReporter/GenerateReportService.ts

```diff
@@ -273,7 +273,9 @@ export class GenerateReportService {
         kanbanSetting.token,
         kanbanSetting.type,
         kanbanSetting.site,
-        kanbanSetting.projectKey,
+        kanbanSetting.type === KanbanEnum.JIRA
```

Comment:
it's better to extract since we use the same logic in 3 places
```
kanbanSetting.type === KanbanEnum.JIRA
          ? kanbanSetting.projectKey
          : kanbanSetting.teamName,
```

Reply:
fixed it into mapping